c7n/manager.py

```diff
@@ -16,12 +16,14 @@
 import logging
 
 from c7n import cache
+from c7n.actions import add_auto_tag_user
 from c7n.executor import ThreadPoolExecutor
 from c7n.registry import PluginRegistry
 from c7n.utils import dumps
 
 
 resources = PluginRegistry('resources')
+resources.subscribe(resources.EVENT_FINAL, add_auto_tag_user)
```

Comment:
i'd rather have the subscription next to the subscriber ie move this to actions.py, its clearer on usage of the subscriber, and avoids potential import cycles, manager is effectively the root of the class tree.